tests/e2e/gc_rwx_syncer.go

```diff
@@ -0,0 +1,866 @@
+		defer func() {
+			if !isControllerUp {
+				bringUpCsiController(svcClient)
+			}
+		}()
+
+		// Create a Pod to use this PVC
+		ginkgo.By("Creating pod to attach PV to the node")
+		pod := fpod.MakePod(namespace, nil, []*v1.PersistentVolumeClaim{pvclaim}, false, execRWXCommandPod1)
+		pod.Spec.Containers[0].Image = busyBoxImageOnGcr
+		pod, err = client.CoreV1().Pods(namespace).Create(ctx, pod, metav1.CreateOptions{})
+		gomega.Expect(err).NotTo(gomega.HaveOccurred(), "Pod creation failed")
+
+		defer func() {
+			// Delete POD
+			ginkgo.By(fmt.Sprintf("Deleting the pod %s in namespace %s", pod.Name, namespace))
+			err = fpod.DeletePodWithWait(client, pod)
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+			ginkgo.By("Verifying whether the CnsFileAccessConfig CRD is Deleted or not for Pod")
+			verifyCNSFileAccessConfigCRDInSupervisor(ctx, f, pod.Spec.NodeName+"-"+pvcNameInSV,
+				crdCNSFileAccessConfig, crdVersion, crdGroup, false)
+		}()
+
+		time.Sleep(oneMinuteWaitTimeInSeconds * time.Second)
+
+		pod, err = client.CoreV1().Pods(namespace).Get(ctx, pod.Name, metav1.GetOptions{})
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		ginkgo.By("Verify Pod is still in ContainerCreating phase")
+		gomega.Expect(podContainerCreatingState == pod.Status.ContainerStatuses[0].State.Waiting.Reason).
+			To(gomega.BeTrue())
+
+		ginkgo.By("Verifying whether the CnsFileAccessConfig CRD is created or not for Pod")
+		framework.Logf("Looking for CnsFileAccessConfig CRD %s", pod.Spec.NodeName+"-"+pvcNameInSV)
+		verifyCNSFileAccessConfigCRDInSupervisor(ctx, f, pod.Spec.NodeName+"-"+pvcNameInSV,
+			crdCNSFileAccessConfig, crdVersion, crdGroup, true)
+
+		ginkgo.By("Bring up csi-controller pod in SV")
+		bringUpCsiController(svcClient)
+		isControllerUp = true
+
+		ginkgo.By("Wait for pod to be up and running")
+		err = fpod.WaitForPodRunningInNamespace(client, pod)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		//Add a check to validate CnsVolumeMetadata crd
+		err = waitAndVerifyCnsVolumeMetadata4GCVol(fcdIDInCNS, pvcNameInSV, pvclaim, persistentvolumes[0], pod)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		ginkgo.By("Verify the volume is accessible and Read/write is possible")
+		cmd := []string{"exec", pod.Name, "--namespace=" + namespace, "--", "/bin/sh", "-c",
+			"cat /mnt/volume1/Pod1.html "}
+		output := framework.RunKubectlOrDie(namespace, cmd...)
+		gomega.Expect(strings.Contains(output, "Hello message from Pod1")).NotTo(gomega.BeFalse())
+
+		writeCmd := []string{"exec", pod.Name, "--namespace=" + namespace, "--", "/bin/sh", "-c",
+			"echo 'Hello message from test into Pod1' > /mnt/volume1/Pod1.html"}
+		framework.RunKubectlOrDie(namespace, writeCmd...)
+		output = framework.RunKubectlOrDie(namespace, cmd...)
+		gomega.Expect(strings.Contains(output, "Hello message from test into Pod1")).NotTo(gomega.BeFalse())
+	})
+
+	/*
+		Annotations should be accessble to all the RWX PVCs which are in bound state.
```

Comment:
the test steps don't mention anything about annotations, what is the test actually trying to do?